processor/src/deadwood_segmentation/unet_model.py

```
@@ -0,0 +1,49 @@
+import torch
+import torch.nn as nn
+
+from .unet_parts import *
```

Ruff (0.8.0)

`from .unet_parts import *` used; unable to detect undefined names

(F403)